cron/update_lunch.py

```diff
@@ -127,25 +138,42 @@ def read_lunches():  # Update the database with new lunches
     add_events(mainresponse)
 
 
-"""
 # Returns lunches to be displayed in a schedule
 def get_lunch_for_date(current_date, days_into_past=28):
-    # days_into_past is the number of days into the past to go
-    earliest_lunch = current_date - datetime.timedelta(days_into_past)
-    query = Lunch.query(Lunch.day >= earliest_lunch)
-    lunch_objs = []
-    for lunch_obj in query:
-        obj = {
-            "summary": lunch_obj.summary,
-            "description": lunch_obj.description,
-            "day": lunch_obj.day.day,
-            "month": lunch_obj.day.month,
-            "year": lunch_obj.day.year,
-        }
-        lunch_objs.append(obj)
+    with client.context():
+        # days_into_past is the number of days into the past to go
+        earliest_lunch = current_date - datetime.timedelta(days_into_past)
+        lunch_objs = []
+        for lunch_obj in Lunch.queryWithTimeConstraint(earliest_lunch):
+            cleaned_description = (
+                []
+            )  # the desc after it is cleaned of escape characters and new lines
+            for description_section in lunch_obj.description:
+                if not (
+                    description_section == ""
+                    or description_section == " "
+                    or description_section == False

```

Comment:
Is it possible for `description_section` to not be a string?

Reply:
was in old version